### PR TITLE
Add additional KB resources

### PR DESCRIPTION
**Pull request by Rezney**
add aditional KB resources in a form of ExternalLink or error details as requested by support

### CODE REVIEW

File: repos/system_upgrade/common/actors/checkdetecteddevicesanddrivers/libraries/checkdddd.py [hunk 1]
```diff
@@ -35,6 +35,16 @@ def create_inhibitors(inhibiting_entries):
                     source=get_source_major_version(),
                 )
             ),
+            reporting.ExternalLink(
```
Comment (mkluson) [inline repos/system_upgrade/common/actors/checkdetecteddevicesanddrivers/libraries/checkdddd.py:38]: I would keep just one link in the tooling and possibly merge content of both solution.

File: repos/system_upgrade/common/actors/persistentnetnamesdisable/actor.py [hunk 2]
```diff
@@ -50,6 +50,11 @@ def process(self):
                     title='How to perform an in-place upgrade to RHEL 8 when using kernel NIC names on RHEL 7',
                     url='https://access.redhat.com/solutions/4067471'
                 ),
+                reporting.ExternalLink(
+                    title='RHEL 8 to RHEL 9: inplace upgrade fails at '
```
Comment (mkluson) [inline repos/system_upgrade/common/actors/persistentnetnamesdisable/actor.py:54]: [nitpicking] usually you put `url` first

File: repos/system_upgrade/common/libraries/rhsm.py [hunk 3]
```diff
@@ -85,7 +85,8 @@ def _handle_rhsm_exceptions(hint=None):
             details={
                 'details': str(e),
                 'stderr': e.stderr,
-                'hint': hint or _def_hint
+                'hint': hint or _def_hint,
+                'link': 'https://access.redhat.com/solutions/6138372'
```
Comment (mkluson) [inline repos/system_upgrade/common/libraries/rhsm.py:89]: no `title`?

File: repos/system_upgrade/el7toel8/actors/checkbtrfs/actor.py [hunk 4]
```diff
@@ -41,6 +41,10 @@ def process(self):
                             title='How do I prevent a kernel module from loading automatically?',
                             url='https://access.redhat.com/solutions/41278'
                         ),
+                        reporting.ExternalLink(
+                            title='Leapp upgrade fail with error "Inhibitor: Btrfs has been removed from RHEL8"',
```
Comment (mkluson) [inline repos/system_upgrade/el7toel8/actors/checkbtrfs/actor.py:45]: `url` is first in majority of cases

**Review by mkluson — COMMENTED** [review]
Do we really want to have all the titles hard-coded in the tooling? I can imagine the titles will change over time and it will lead to unnecessary confusion. I would be ok to have just plain links.

File: repos/system_upgrade/common/actors/persistentnetnamesdisable/actor.py (hunk 2 above)
Comment (pirat89) [reply repos/system_upgrade/common/actors/persistentnetnamesdisable/actor.py:54]: it's ok, the order is not important in this case. let's keep it

File: repos/system_upgrade/common/libraries/rhsm.py (hunk 3 above)
Comment (pirat89) [reply repos/system_upgrade/common/libraries/rhsm.py:89]: it's a detail for an error msg, not report. so not title.

**Review by pirat89 — APPROVED** [review]
lgtm. merging

**Comment by pirat89** [comment]
@andywaltlova I guess this one is not important for you, as only external links have been added and no other change in reports happened. but to make it simple, setting the report label too.

**Pull request merged**